settings.gradle

```diff
@@ -7,6 +7,7 @@ pluginManagement {
     id "io.github.gradle-nexus.publish-plugin" version "1.0.0"
     id "me.champeau.jmh" version "0.6.4"
     id "net.ltgt.errorprone" version "1.3.0"
+    id 'org.jetbrains.kotlin.jvm' version '1.5.10'
```

Comment:
Kotlin version needs to match gradle plugin version, so figured best to just manage it same as other plugins